packages/scss/src/utilities/_spacing.scss

```diff
@@ -10,105 +10,105 @@
 /* Margin bottom */
 
```

Comment:
But are they useful when you have people dedicated to styling ? Isn't is risky to offer such tools that can turn into "inline styling" ?